grafana-plugin/src/components/Policy/NotificationPolicy.tsx

```diff
@@ -149,13 +152,15 @@ export class NotificationPolicy extends React.Component<NotificationPolicyProps,
   }
 
   _renderTelegramNote() {
-    const { telegramVerified } = this.props;
-
-    return telegramVerified ? (
-      <PolicyNote type="success">Telegram is connected</PolicyNote>
-    ) : (
-      <PolicyNote type="danger">Telegram is not connected</PolicyNote>
-    );
+    const { telegramVerified, store } = this.props;
+
+    return store.hasFeature(AppFeature.Telegram) ? (
+      telegramVerified ? (
+        <PolicyNote type="success">Telegram is connected</PolicyNote>
+      ) : (
+        <PolicyNote type="danger">Telegram is not connected</PolicyNote>
+      )
+    ) : null;
```

Comment:
May I ask if possible to put the null condition out first?

```jsx
if (!store.hasFeature(AppFeature.Telegram)) return null;

return telegramVerified ? (
    <PolicyNote type="success">Telegram is connected</PolicyNote>
  ) : (
    <PolicyNote type="danger">Telegram is not connected</PolicyNote>
  )
```

Slightly easier to read than 2 nested conditions